### PR TITLE
fix(api): browser enabled pullpath

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: packages/buckets/src/buckets.spec.ts [hunk 1]
```diff
@@ -210,27 +215,41 @@ describe('Buckets...', function () {
...
       const chunks = client.pullPath(rootKey, 'dir1/file1.jpg', {
         progress: (num) => (length = num || 0),
       })
-      const pth = path.join(__dirname, '../../..', 'testdata')
-      const stream = fs.createWriteStream(path.join(pth, 'output.jpg'))
-      // Should be an AsyncIterable
-      for await (const chunk of chunks) {
-        stream.write(chunk)
+      if (isBrowser) {
+        let result = new Uint8Array();
```
Comment (andrewxhill) [inline packages/buckets/src/buckets.spec.ts:226]: uploaded file is different so has a different set of expectations

File: packages/buckets/src/buckets.spec.ts [hunk 2]
```diff
@@ -159,7 +164,7 @@ describe('Buckets...', function () {
       const res = await client.pushPath(rootKey, 'dir1/file1.jpg', file, {
         progress: (num) => (length = num || 0),
       })
-      expect(length).to.equal(54)
+      expect(length).to.equal(620047)
```
Comment (andrewxhill) [inline packages/buckets/src/buckets.spec.ts:167]: used a much larger test input string to ensure we span more chunks

File: packages/buckets/src/buckets.spec.ts [hunk 3]
```diff
@@ -210,27 +215,41 @@ describe('Buckets...', function () {
...
         progress: (num) => (length = num || 0),
       })
-      const pth = path.join(__dirname, '../../..', 'testdata')
-      const stream = fs.createWriteStream(path.join(pth, 'output.jpg'))
-      // Should be an AsyncIterable
-      for await (const chunk of chunks) {
-        stream.write(chunk)
+      if (isBrowser) {
+        let result = new Uint8Array();
+        const append = (target: Uint8Array, addition: Uint8Array) => {
```
Comment (carsonfarmer) [inline packages/buckets/src/buckets.spec.ts:227]: If we don't have this in a util module somewhere, we probably should. I'd guess we use this pattern elsewhere.

**Review by carsonfarmer — APPROVED** [review]
Yeh looks good @andrewxhill , nice to have some more browser-specific tests in there for sure.

File: packages/buckets/src/buckets.spec.ts (hunk 3 above)
Comment (andrewxhill) [reply packages/buckets/src/buckets.spec.ts:227]: with all the packages moving together, it seems like a nice enhancement in the near future will be:

1. a unified set of utils that users might want across the board
2. a unified set of utils used in tests all over the place

**Pull request merged**